app/controllers/api/v2/roles_controller.rb

```diff
@@ -57,7 +57,12 @@ def destroy
       param_group :role
       def clone
         @role = @role.clone(role_params)
-        process_response @role.save
+        if @role.save
+          process_success
+        else
+          @role.add_tax_errors
```

Comment:
Could you move it to `after_validation` callback of `Role`? I guess this error can appear during other operations like update too. Moving it there would cover it.